src/main/java/link/servr/servrlink/velocity/VelocityServerLink.java

```diff
@@ -0,0 +1,112 @@
+package link.servr.servrlink.velocity;
+
+import com.google.inject.Inject;
+import com.moandjiezana.toml.Toml;
+import com.velocitypowered.api.event.Subscribe;
+import com.velocitypowered.api.event.proxy.ProxyInitializeEvent;
+import com.velocitypowered.api.plugin.Plugin;
+import com.velocitypowered.api.plugin.annotation.DataDirectory;
+import com.velocitypowered.api.proxy.ProxyServer;
+import link.servr.servrlink.core.Core;
+import link.servr.servrlink.core.ServrLink;
+import link.servr.servrlink.velocity.commands.LinkCommand;
+import lombok.Getter;
+
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.logging.Logger;
+
+@Plugin(id = "velocityserverlink", name = "ServrLink", version = "1.0-SNAPSHOT",
+        description = "I did it!", authors = {"TheCoolOne123", "TCOOfficiall", "ryan."})
```

Comment:
Change description

Reply:
Also use github names, at least for me. Perhaps limit yourself to one name as well, instead of including 2 aliases